.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
+.idea/
```

Comment:
Use global gitignore instead, no need to add extra files

Reply:
Where is the global gitignore?

Reply:
I removed local's project gitignore to use global on my machine